Added sponsor denkwerk

agorilla opened the pull request:
- Added SVG logo, text and link
- Slight layout changes to sponsor cards

agorilla committed:
Added sponsor denkwerk

- Added SVG logo, text and link
- Slight layout changes to sponsor cards

agorilla commented:
Please have a look at the implementation.

anselmh commented:
fix that? Then it’s good. ![screen shot 2015-07-02 at 08 51 33](https://cloud.githubusercontent.com/assets/399345/8471358/a5d697ac-2097-11e5-84c3-2932f959ca15.png)

agorilla committed:
Added space between sponsor cards

- 0.5em space top and bottom
- Fixed alignment bug because <p> tag for sponsor subline was missing
- Changed cards <p> strong color to color to text color instead of green (was not readable in sponsor cards)

agorilla commented:
Fixed mobile bug
